File: 1_starter/java/src/main/java/no/systek/graphqlworkshop/storage/PersonRepository.java
package no.systek.graphqlworkshop.storage;

import java.util.ArrayList;
import java.util.List;
import org.springframework.stereotype.Repository;

@Repository
public class PersonRepository {

  private static ArrayList<Person> exampleData = new ArrayList<Person>() {{
    add(new Person("Karl", "Frontend Developer"));
    add(new Person("Aage", "Software Architect"));
    add(new Person("Ole", ".NET Developer"));
  }};

  public List<Person> getAll() {
    return exampleData;
  }
}

File: 1_starter/kotlin/src/main/kotlin/no/systek/graphqlworkshop/storage/PersonRepository.kt
package no.systek.graphqlworkshop.storage

import org.springframework.stereotype.Repository

@Repository
class PersonRepository {
    private val exampleData = listOf(
        Person("Karl", "Frontend Developer"),
        Person("Aage", "Software Architect"),
        Person("Mikael", "Typer of Code")
    )

    fun getAll() = exampleData
}
